lib/steps/load_state.go

```diff
@@ -5,18 +5,41 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"os"
 )
 
-func loadState(command string) RunState {
+func hasSavedState(command string) bool {
 	filename := getRunResultFilename(command)
-	content, err := ioutil.ReadFile(filename)
+	_, err := os.Stat(filename)
 	if err != nil {
-		log.Fatal(err)
+		if os.IsNotExist(err) {
```

Comment:
go doesn't let that compile. It errors with: `lib/steps/load_state.go:21: missing return at end of function`